app/presenters/summary/components/date_time_answer.rb

```diff
@@ -0,0 +1,9 @@
+module Summary
+  module Components
+    class DateTimeAnswer < BaseAnswer
+      def to_partial_path
+        'steps/submission/shared/date_time_answer'
```

Comment:
Cool I see what you tried to do, and this will totally work. But we can take advantage of the fact the `BaseAnswer` superclass is already prepared to take i18n options, and propagate them to the component.

So we can keep using the existing `DateTime` component, and we pass `i18n_opts: { format: :datetime }`.
There is a tiny change needed in the partial to use these i18n opts:

`l(date_answer.value, **date_answer.i18n_opts)`

And finally a little scenario in the view test here to double check it works as expected 😄 
`spec/views/steps/submission/shared/_date_answer.html.erb_spec.rb`